evm_loader/deploy-test.sh

```diff
@@ -15,7 +15,7 @@ solana airdrop 1000
 solana account $ACCOUNT
 
 echo "Run tests for EVM Loader"
-export EVM_LOADER=$(solana-deploy deploy evm_loader.so | tail -n 1 | python3 -c 'import sys, json; data=json.load(sys.stdin); print(data["programId"]);')
+export EVM_LOADER=$(solana-deploy deploy evm_loader.so | sed '/Program Id: \([0-9A-Za-z]\+\)/,${s//\1/;b};s/^.*$//;$q1')
```

Comment:
Maybe it would be better to add a comment above with the example of a source string 
If the source format will change than we can easily change the extraction code of sed

Reply:
Fixed.